src/Changelog.js

```diff
@@ -40,60 +40,57 @@ export default class Changelog {
     }).forEach(category => {
       progressBar.tick(category.heading);
 
-      markdown += "\n";
-      markdown += "\n";
-      markdown += "#### " + category.heading;
+      const commitsByPackage = {};
 
       category.commits.forEach(commit => {
-        markdown += "\n";
-
-        var changedPackages =
-        execSync("git log -m --name-only --pretty='format:' " + commit.commitSHA)
-        // not sure why it's including extra files
-        .split("\n\n")[0]
-        // turn into an array
-        .split("\n")
-        // remove files that aren't in packages/
-        .filter(function(files) {
-          return files.indexOf("packages/") >= 0;
-        })
-        // extract base package name
-        .map(function(files) {
-          files = files.slice(9);
-          return files.slice(0, files.indexOf("/"));
-        })
-        // unique packages
-        .filter(function(value, index, self) {
-          return self.indexOf(value) === index;
-        });
 
-        var spaces = 0;
+        // Array of unique packages.
+        var changedPackages = Object.keys(
+          execSync("git show -m --name-only --pretty='format:' --first-parent " + commit.commitSHA)
+          // turn into an array
+          .split("\n")
+          // extract base package name, and stuff into an object for deduping.
+          .reduce(function(obj, files) {
+            if (files.indexOf("packages/") === 0) {
+              obj[files.slice(9).split("/", 1)[0]] = true;
+            }
+            return obj;
+          }, {})
+        );
+
+        const heading = changedPackages.length > 0
+          ?"* "+changedPackages.map(pkg => "`" + pkg + "`").join(", ")
+          :"* Other"; // No changes to packages, but still relevant.
```

Comment:
This is worth mentioning: Previously changes with no associated packages bumped out to the outer list level.  This branch moves them into an "Other" group.  These are changes that get assigned to a changelog category but don't affect any packages.